src/OaiPmh/Harvester.php

```diff
@@ -168,10 +172,38 @@ public function launch()
             $sets = [null];
         }
 
+        // The harvestEndDate may be null. Some OAI-PMH hosts may depend on a
+        // null value for backwards compatibility and reliability for various
+        // edge cases, so we allow a null value to be used during the initial
+        // records request. However, we still need to track an explicit end
+        // date, based on the current host time, as the basis for future
```

Comment:
I noticed that you changed "server" to "host" in your last commit... but I thought "server" seemed more accurate. Maybe we should say "OAI server" to be more explicit.

Reply:
Yeah, I think we have matched intent here. I want to make clear that the time is not based on local VuFind environment. I'll make edit to make this clear.